Modelplace_Cloud_API/api_requests.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import json
+
+
+def login_request(email: str, password:str) -> json:
+    response = requests.post(
+        url="https://api.modelplace.ai/v3/login", 
```

Comment:
Specify the global variable `API = https://api.modelplace.ai/v3/` and add subpaths to that os.path.join(API, "login")